Cargo.toml

```diff
@@ -66,8 +66,9 @@ p384 = { version = "0.13.0", optional = true }
 rsa = { version = "0.9.2", optional = true }
 sha2 = { version = "0.10.8", optional = true }
 x509-cert = { version = "0.2.4", optional = true }
+byteorder = "1.4.3"
 
 [dev-dependencies]
 kvm-ioctls = ">=0.12"
 kvm-bindings = ">=0.6"
-serial_test = "2.0"
+serial_test = "2.0"
```

Comment:
Not sure if this is an artifact of the merge conflict. Maybe double-check with main before adding this back in.